js/ui/grid_core/ui.grid_core.utils.js

```diff
@@ -162,7 +169,7 @@ module.exports = (function() {
             startFilterExpression = [dataField, ">=", filterValue[0]];
             endFilterExpression = [dataField, "<=", filterValue[1]];
 
-            if(column.dataType === "date") {
+            if(module.exports.isDateType(column.dataType)) {
```

Comment:
Save "module.exports.isDateType" to variable because isDateType is used in several places